docs/navbar.js

```diff
@@ -35,6 +39,16 @@ function loadNavbarWithPicker(firebaseAuthPromise) {
   loadNavbar(
       () => $('#nav-left')
                 .load(
-                    '/disaster_picker.html',
+                    getUrlUnderDocs('disaster_picker.html'),
                     () => initializeDisasterPicker(firebaseAuthPromise)));
 }
+
+/**
+ * Get url of a file in or below our docs directory by using the path of this
+ * current script.
+ * @param {string} replaceWith
```

Comment:
I feel like a better name might just be `path` right?

Reply:
oh yes, I renamed this method and forgot to appropriately rename the param. thanks!